getting-started/vs-code-integration/convert-project-wizard.md

```diff
@@ -27,26 +27,20 @@ To add the Telerik Components to your existing Blazor application, use the `Conv
 
 ## Convert your project
 
-1. Press `Ctrl`+`Shift`+`P` in Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open VSCode's extension launcher
+1. Press `Ctrl`+`Shift`+`P` on Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open the VS Code extension launcher.
 
-![Launch Convert Command](images/launch-convert-command.png)
+   ![Launch Convert Command](images/launch-convert-command.png)
 
-1. Type/Select `Telerik UI for Blazor Convert Command: Launch` and press `Enter` to launch the convert command
+1. Type/Select `Telerik UI for Blazor Convert Command: Launch` and press `Enter` to launch the convert command. This will start the conversion process. The following notification will appear in the bottom right corner of the VS Code window to indicate that the conversion has successfully passed:
 
-1. The conversion process will start
+   ![Success Notification](images/successfully-converted-example.png)
 
-
-1. The following notification will appear in the bottom right corner of the VS Code to indicate that the conversion has successfully passed.
-
-![Success Notification](images/successfully-converted-example.png)
-
-
-If the Telerik components are already enabled in the project you will see the following notification:
+If the Telerik components are already enabled in the project, you will see the following notification:
 
 ![Already converted project Notification](images/already-converted-example.png)
 
 
->tip You now have enabled the Progress&reg; Telerik&reg; UI for Blazor components in your Blazor app!
```

Comment:
This doesn't sound like additional information that needs to be visually separated from the rest of the text.